File: src/main/java/EKIDS/Lessons/Lesson09/SWING/DrawSimpleFont.java
package EKIDS.Lessons.Lesson09.SWING;


import java.awt.Dimension;
import java.awt.Font;
import java.awt.Graphics;
import java.awt.Graphics2D;
import java.awt.Toolkit;
import javax.swing.JComponent;
import javax.swing.JFrame;
import javax.swing.WindowConstants;

public class DrawSimpleFont {
    public static void main(String[] args) {
        JFrame jFrame0 = getFrame();
        jFrame0.add(new MyComponent());
    }

    static class MyComponent extends JComponent {
        @Override
        protected void paintComponent(Graphics g) {
            Font font = new Font("Copperplate Gothic Bold", Font.ITALIC, 25);
            Graphics2D g2 = (Graphics2D) g;
            g2.setFont(font);
            g2.drawString("Hello Java", 25, 25);

        }
    }

    static JFrame getFrame() {
        JFrame jFrame = new JFrame();
        jFrame.setVisible(true);
        jFrame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
        Toolkit toolkit = Toolkit.getDefaultToolkit();
        Dimension dimension = toolkit.getScreenSize();
        jFrame.setBounds(dimension.width / 2 - 320, dimension.height / 2 - 240, 640, 480);
        return jFrame;
    }

}

File: src/main/java/EKIDS/Kotlin/Lessons/Lesson10/SWING/DrawSimpleFont.kt
package Kotlin.Lessons.Lesson10.SWING

import java.awt.Font
import java.awt.Graphics
import java.awt.Graphics2D
import java.awt.Toolkit
import javax.swing.JComponent
import javax.swing.JFrame
import javax.swing.WindowConstants

object DrawSimpleFont {
    @JvmStatic
    fun main(args: Array<String>) {
        val jFrame0 = frame
        jFrame0.add(MyComponent())
    }

    val frame: JFrame
        get() {
            val jFrame = JFrame()
            jFrame.isVisible = true
            jFrame.defaultCloseOperation = WindowConstants.EXIT_ON_CLOSE
            val toolkit = Toolkit.getDefaultToolkit()
            val dimension = toolkit.screenSize
            jFrame.setBounds(dimension.width / 2 - 320, dimension.height / 2 - 240, 640, 480)
            return jFrame
        }

    internal class MyComponent : JComponent() {
        override fun paintComponent(g: Graphics) {
            val font = Font("Copperplate Gothic Bold", Font.ITALIC, 25)
            val g2 = g as Graphics2D
            g2.font = font
            g2.drawString("Hello Kotlin", 25, 25)
        }
    }
}